api/src/main/java/io/opencensus/trace/propagation/PropagationComponent.java

```diff
@@ -49,6 +49,19 @@
   @ExperimentalApi
   public abstract TextFormat getB3Format();
 
+  /**
+   * Returns the B3 {@link TextFormat} with the provided implementations. See <a href=
```

Comment:
maybe

```
 Returns the {@link TextFormat B3 single text format}.
```

Reply:
As I started modifying the Spring contrib classes to support "single" as a configuration parameter, I realised it's probably better to expose a `boolean` parameter here that controls whether or not to emit trace data in the compact format. What do you think?